website/docs/components/badge/index.md

```diff
@@ -1,14 +1,22 @@
 ---
 title: Badge
-description: Concise, non-interactive labels that represent metadata.
-caption: Concise, non-interactive labels that represent metadata.
+description: 'Concise, non-interactive labels that represent metadata.'
```

Comment:
Question: why do these caption and description have quotes? it seems like most of the others dont.

Reply:
Looking at the deployed site (https://hds-website-git-alex-ju-automate-component-ver-2d7be8-hashicorp.vercel.app/components/badge) seems like it doesn't randomly add the quotes to the page so I guess its fine

Reply:
According to the schema used by `js-yaml` strings over a certain length are moved to a block (hence the `>-` notation and the short ones get wrapped in single quotes to avoid parse errors.

Reply:
Ok! I just noticed that there were places where the file was updated, but caption/description stayed the same (didn't add the quotes) so I thought it was interesting that it only got added 3 times.

ex: Accordion didnt have quotes added around the caption
![Screenshot 2025-01-22 at 10 10 58 AM](https://github.com/user-attachments/assets/143a8774-af17-4bdf-b95e-dfbd55c21191)